redash/destinations/discord.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+import requests
+
```

Comment:
There add this
`from redash.models import Alert`

Reply:
Done.